instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb

```diff
@@ -19,13 +19,14 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         present do
           !defined?(::ActiveSupport::Notifications).nil? &&
             !defined?(::ActiveModel::Serializers).nil? &&
```

Comment:
ActiveModel actually defines this namespace without the presence of the `active_model_serializers` gem 

https://github.com/rails/rails/blob/master/activemodel/lib/active_model.rb#L62